src/landscape/components/LandscapeBoundaries.test.js

```diff
@@ -139,13 +137,9 @@ test('LandscapeBoundaries: Select file', async () => {
     },
   };
   fireEvent.drop(dropzone, data);
-  await waitFor(() =>
-    expect(
-      screen.getByRole('button', {
-        name: 'Select File Accepted file formats: *.json, *.geojson File size limit: 1MB test.json 0.8KB',
-      })
-    ).toBeInTheDocument()
-  );
+  await screen.findByRole('button', {
```

Comment:
Question: same question here

Reply:
This wasn't a change I intended. Maybe a merge conflict?

Reply:
Looks like `npm run lint-js` makes this change.